pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/loader/defaultcolumn/BaseDefaultColumnHandler.java

```diff
@@ -402,6 +450,12 @@ private void createDefaultValueColumnV1Indices(String column)
     boolean isSingleValue = fieldSpec.isSingleValueField();
     int maxNumberOfMultiValueElements = isSingleValue ? 0 : 1;
 
+    // TODO: These validations are done as a sanity check though the reload code doesn't actually look at the
```

Comment:
Does "TODO" refer to items that still need to be resolved in this PR or do they refer to possible future work items that are not part of this PR? Perhaps these comments can be reworded a bit as the "TODO" gives the impression that something has been left in an unresolved state or maybe opening a github issue with a list of future items might be more useful?

Reply:
This will be addressed in the next PR to handle reload, I wanted to keep the checks to ensure that someone doesn't enable this feature while using some incompatible configs, but actual handling for default column will be the next related PR. Reworded this comment